README.md

```diff
@@ -30,12 +30,32 @@ Among other things, it:
 - Concatenates the files from `paella-matterhorn/javascript` and `paella-matterhorn/plugins` into `paella_matterhorn.js`.
 - Concatenates localization json files together.
 
+*How to read the Makefile*
+
+[Make](https://bost.ocks.org/mike/make/) is a venerable and reliable Unix build tool. The Makefile format lets you specify targets that consist of:
+
+1. Unix commands
+2. Dependencies (other targets)
+
+For example, the `copy-extensions-to-paella` target does nothing itself but depends on five other targets, like `copy-vendor-extensions-to-paella`. `copy-vendor-extensions-to-paella` creates a `vendor` directory under the Paella submodule directory if it doesn't exist and copies files from `dce-paella-extensions` into it.
+
+    copy-extensions-to-paella: \
+      copy-vendor-extensions-to-paella \
+      copy-resources-to-paella \
+      copy-test-repository-to-paella \
+      copy-config-to-paella \
+      copy-skins-to-paella
+
+    copy-vendor-extensions-to-paella:
+      mkdir -p $(PAELLADIR)/vendor && \
+      cp -r $(EXTDIR)/vendor/* $(PAELLADIR)/vendor
+
 Dependencies
 ------------
 
-dce-paella-extensions is a dependency that contains DCE-specific Paella changes. Its version is specified as a tilde range, which means that NPM will install the latest version of dce-paella-extensions that matches the major and minor versions. e.g. If ~1.0.27 is the version in package.json, NPM will install the 1.0.30 if it is available. It will not install 1.1.0. You need to change the version in package.json if you want that version.
+dce-paella-extensions is a dependency that contains DCE-specific Paella changes.
 
-This project uses [NPM Shrinkwrap](https://docs.npmjs.com/cli/shrinkwrap). It installs modules from the `npm-shrinkwrap.json` file, not the `package.json` file and will install the exact versions of the dependencies and subdependencies defined in `npm-shrinkwrap.json`. If you want to update dependencies, you must run `npm install --save <module name>@<version>` or `npm update --save <module name>` in order to update both `package.json` and `npm-shrinkwrap.json`.
+This project uses [NPM Shrinkwrap](https://docs.npmjs.com/cli/shrinkwrap). It installs modules from the `npm-shrinkwrap.json` file, not the `package.json` file and will install the exact versions of the dependencies and subdependencies defined in `npm-shrinkwrap.json`. If you want to update dependencies, you must run `npm install --save <module name>@<version>` in order to update both `package.json` and `npm-shrinkwrap.json`.
```

Comment:
1. We did. In fact, we found out because changes to `package.json` alone weren't being observed. (Because `npm-shrinkwrap.json` supersedes it.)
2. The `module name` is the name of the dependency module you want to update. For example, dce-paella-extensions. You only run just plain `npm shrinkwrap` to generate `npm-shrinkwrap.json` initially. I'll put this in there.